uwg_2023/uwg_2023_burst_demo.ipynb

```diff
@@ -0,0 +1,324 @@
+   "source": [
+    "## 3. Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c98292cf-7721-464b-807f-33da6bd15712",
+   "metadata": {},
+   "source": [
+    "Follow the developer setup instructions in the [HyP3-ISCE2 readme](https://github.com/asfhyp3/hyp3-isce2#developer-setup), then run the command below:\n",
+    "\n",
+    "```bash\n",
+    "mamba activate hyp3-isce2\n",
+    "mamba install -y jupyter jupyterlab hyp3_sdk\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7a02761b-0384-4490-aa93-69ba5f8a7544",
+   "metadata": {},
+   "source": [
+    "## 4. Local Python Package Interface"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8bb6bdb9-baac-4220-8bc0-d5cbe88c2f38",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir local_run"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9c9b8424-7d38-4667-8831-8913b1a2a9f8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cd local_run"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f7e43296-a265-4be3-aeec-f82e05695da5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -m hyp3_isce2 ++process insar_tops_burst \\\n",
+    "  S1_136231_IW2_20200604T022312_VV_7C85-BURST \\\n",
+    "  S1_136231_IW2_20200616T022313_VV_5D11-BURST \\\n",
+    "  --looks 20x4"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "53fa0110-1e3a-42b9-b794-31a9465490fe",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cd .."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "84a049a2-49af-42d5-8e7a-43d48371dd93",
+   "metadata": {},
+   "source": [
+    "## 4. HyP3 Interface"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "95e06a19-614c-4b7e-8254-bc251cba964e",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdin",
+     "output_type": "stream",
+     "text": [
+      "NASA Earthdata Login username:  ffwilliams2\n",
+      "NASA Earthdata Login password:  ········\n"
+     ]
+    }
+   ],
+   "source": [
+    "import copy\n",
+    "import hyp3_sdk as sdk\n",
+    "job_definition = {\n",
+    "    'job_parameters': {\n",
+    "        \"granules\": [\n",
+    "            \"S1_136231_IW2_20200604T022312_VV_7C85-BURST\",\n",
+    "            \"S1_136231_IW2_20200616T022313_VV_5D11-BURST\"\n",
+    "        ],\n",
+    "        \"looks\": \"20x4\"\n",
+    "    },\n",
+    "    \"job_type\": \"INSAR_ISCE_BURST\",\n",
+    "    \"name\": \"UWG_BURST\"\n",
+    "}\n",
+    "hyp3 = sdk.HyP3(api_url='https://hyp3-enterprise-test.asf.alaska.edu', prompt=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0379879d-c7ed-41cf-9f92-757fe4afcc1d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "jobs = sdk.Batch()\n",
+    "job = copy.deepcopy(job_definition)\n",
+    "jobs += hyp3.submit_prepared_jobs(job)"
```

Comment:
`submit_prepared_jobs` accepts a single job dictionary, and returns a Batch, so unless I'm missing something this can reduce to:
```suggestion
    "jobs = hyp3.submit_prepared_jobs(job)"
```